README.md

```diff
@@ -336,15 +336,40 @@ Or run the following example that evaluates trials on SageMaker.
 python examples/launch_height_sagemaker.py
 ```
 
-If you want to launch HPO remotely and not just on your machine, you will also need `docker` installed on your local
-machine, see [this link](https://docs.docker.com/get-docker/) to install it on your machine.
-The following example should then work which will schedule the tuning on SageMaker rather than on your machine:
+This example does not require you to build a docker image, since the training code runs
+in a SageMaker framework. However, there are working modes that require building an image, as
+we discuss next.
+
+First, Syne Tune allows you to launch HPO experiments remotely on SageMaker, instead of them
+running on your local machine. This is particularly interesting for running many
+experiments in parallel. Here is an example:
 ```bash
 python examples/launch_height_sagemaker_remotely.py
 ```
+If you run this for the first time, it will take a while, building a docker image with the
+Syne Tune dependencies and pushing it to ECR. This has to be done only once, even if Syne
+Tune source code is modified later on.
+
+If running this example script fails, you are probably not setup to build docker images and
+push them to ECR on your local machine. In order to enable this, you need to install both
+`awscli` and `docker` (see [this link](https://docs.docker.com/get-docker/)),
+and make sure the docker demon is running. However, since many Syne Tune workflows
+do not require building docker images, you may alternatively build the required image
+on a SageMaker notebook instance. To do so, log into such an instance and install
+Syne Tune there, then run:
+```bash
+cd container
+bash build_syne_tune_container.sh
+```
```

Comment:
Can we remove the part after "However, ...". Running in a SageMaker notebook instance is not a unit step (we would have to describe all the steps precisely), also I do not think it is simpler than running a docker daemon locally.

Reply:
OK, but can I add a comment in parentheses, saying that awscli and docker will be all installed on a notebook instance?

I am really thinking about our customers here. Many will be using notebooks instances anyway, exactly because they don't want to install things like docker.

What I am describing here is exactly something that affects me right now. I cannot reinstall docker, because it does not support my system anymore (I got the laptop maybe 2 years ago). And linking the docker main page is not a complete description on how to get docker running. On Mac OS, starting the docker daemon is really quite tricky, for example.

Reply:
Ok sure, my point is that for SageMaker users will know how to set a SM notebook but probably not others so this comment might confuse them. I think SageMaker often assume that you have a docker setup somehow.

Not sure why docker would not run on your machine, it should be supported for (much) older machine, lets discuss this one offline.